docs/administration/user-permissions.md

```diff
@@ -0,0 +1,108 @@
+# Users and permissions
+
+In a funding program and for a specific funding case, different people from the giving organisation might be involved. If the receiving contact is an organisation, there might be different people interacting with the funding case as well. These people have different roles that come with different permissions.
+
+The permissions need to be configured on different levels:
+
+* Drupal permissions
+* CiviRemote User roles on a CiviCRM contact summary page
+* Funding program permissions
+* Allowed recipients for a funding program
+* Funding Case permissions
+
+Some permissions need to be configured only once at the initial configuration of the funding extension. Others have to be configured during the creation of a new funding program. Some have to be configured for every individual contact.
+
+## Drupal permissions
+
+After installing the funding extension, the Drupal roles need to be given the correct permissions. You can edit the Drupal permissons at **People > Permissions** or `admin/people/permissions`, respectively.
+
+* Grant the following permissions to the already existing role *authenticated user*:
+    * Access CiviCRM Funding
+    * Access fontawesome additional settings
```

Comment:
I don't know. I tried to find the differences between the predefined permissions of my test installation and the permissions of our two demo systems. This is what I found. I think `Access CiviCRM Funding` is important for the role `CiviRemote: CiviRemote Funding` (as you corrected below). I remember that it initially didn't work without this permission.
I set the fontawsome permisson only because I found this to be a difference as well. Shall I remove it from the text?

Reply:
Yes, please remove `Access fontawesome additional settings`. We don't want to give any authenticated user access to internal settings.